hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/handlers/FreezeHandler.java

```diff
@@ -167,7 +165,7 @@ public void handle(@NonNull final HandleContext context) throws HandleException
         final var filesConfig = context.configuration().getConfigData(FilesConfig.class);
 
         final FreezeUpgradeActions upgradeActions = new FreezeUpgradeActions(
-                adminServiceConfig, freezeStore, freezeExecutor, upgradeFileStore, nodeStore, stakingInfoStore);
+                context, freezeStore, freezeExecutor, upgradeFileStore, nodeStore, stakingInfoStore);
```

Comment:
Seems like context is not needed

Reply:
Yes, changed to use Configuration, good suggest. Thanks @Neeharika-Sompalli